src/coreclr/tools/StressLogAnalyzer/StressLogDump.cpp

```diff
@@ -80,14 +95,15 @@ const char *getFacilityName(DWORD_PTR lf)
         #include "../../../inc/loglf.h"
         { LF_ALWAYS, "LF_ALWAYS" }
     };
-    static char buff[1024] = "`";
+    const int buff_length = 1024;
```

Comment:
A nit - I wonder if you have used the `buff_length` for something before and then changed stuff. It seems unnecessary when it is used just to create the array. Similar for the `format_length` and `name_buffer_length` below.